src/plugins/router.js

```diff
@@ -46,6 +47,8 @@ module.exports = {
       ],
     })
 
+    addKeywords(ajv)
```

Comment:
`addKeywords` (and also `addFormats`) should have a minimal set of defaults and allow users to add more from `.env`, but this is for another time.

At first glance at [ajv-keywords](https://github.com/ajv-validator/ajv-keywords#contents), I would go with `transform` and `dynamicDefaults`.